spec/lib/backend_error_spec.rb

```diff
@@ -483,127 +483,310 @@
     end
 
     it "returns empty for scheduled messages" do
```

Comment:
empty?

Reply:
Sorry, @nekron, I've forgotten to rename this test. It returned empty before these changes. I'll rename it in the next pull request. Thanks!